api/desecapi/tests/base.py

```diff
@@ -545,6 +546,12 @@ def assertResponse(self, response, code=None, body=None):
         if body:
             self.assertJSONEqual(response.content, body)
 
+    def assertToken(self, plain, user=None):
+        user = user or self.owner
+        hashed = make_password(plain, salt='static', hasher='pbkdf2_sha256_iter1')
+        self.assertEqual(len(hashed.split('$')), 4)
+        self.assertIn(hashed, Token.objects.filter(user=user).values_list('key', flat=True))
```

Comment:
can we use Django's check mechanism here?

Reply:
good catch! See fix in https://github.com/desec-io/desec-stack/commit/9774af98ee8dab78e5df2a3a35169fdb18ea276c

Downsides:
- It's now not so easy anymore to check that hashed tokens in fact are in hashed format, with `$` in it etc, as we're not dealing with a hashed token directly anymore
- The hash is computed in `check_password()` once per candidate, so the same thing is computed repeatedly (but it's very fast, so no problem)